src/lib.rs

```diff
@@ -46,7 +46,7 @@ pub enum RandomXFlag {
 }
 
 impl RandomXFlag {
-    pub fn value(self) -> c_uint {
+    pub fn value(self) -> u32 {
```

Comment:
This value gets passed to C, while equivalent at present is u32 it is probably best left as a c_uint

Reply:
I see every where it gets passed to randomX, it gets cast as a c_uint first. 

From the rust docs: 
```
Equivalent to C's unsigned int type.

This type will almost always be u32, but may differ on some esoteric systems. The C standard technically only requires that this type be an unsigned integer with the same size as an int; some systems define it as a u16, for example.
```

So I think if we keep it a u32 on our side, and then casting it to c_unit before we use it, it should be safe everywhere.